systemtest/src/main/java/io/strimzi/systemtest/resources/keycloak/SetupKeycloak.java

```diff
@@ -103,6 +104,16 @@ private static void deployKeycloak(String namespaceName) {
 
     private static void deployPostgres(String namespaceName) {
         LOGGER.info("Deploying Postgres into Namespace: {}", namespaceName);
+
+        try {
+            final String postgresYaml = Files.readString(Paths.get(POSTGRES_FILE_PATH)).replace(
+                "${POSTGRES_IMAGE}", Environment.POSTGRES_IMAGE
+            );
+            Files.writeString(Paths.get(POSTGRES_FILE_PATH), postgresYaml);
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to update the Postgres deployment YAML", e);
+        }
+
         cmdKubeClient(namespaceName).apply(POSTGRES_FILE_PATH);
```

Comment:
isn't possible to load the content, change the value and apply it without storing it? I think we did it in several cases

Reply:
Yeah, I think I can replace it (which returns a modified YAML). I will update this accordingly.